test/integration_strict_test.go

```diff
@@ -28,29 +28,22 @@ func TestStrictMode(t *testing.T) {
 			name:           "plan-all",
 			controls:       []string{},
 			strictMode:     false,
-			expectedStderr: "The `plan-all` command is deprecated and will be removed in a future version. Use `terragrunt run-all plan` instead.",
+			expectedStderr: "`plan-all` commands is deprecated and will be removed in a future version. Use `terragrunt run-all plan` instead.",
 			expectedError:  nil,
 		},
 		{
 			name:           "plan-all with plan-all strict control",
-			controls:       []string{"plan-all"},
+			controls:       []string{"deprecated-commands"},
 			strictMode:     false,
 			expectedStderr: "",
-			expectedError:  strict.StrictControls[strict.PlanAll].Error,
-		},
-		{
-			name:           "plan-all with multiple strict controls",
-			controls:       []string{"plan-all", "apply-all"},
-			strictMode:     false,
-			expectedStderr: "",
-			expectedError:  strict.StrictControls[strict.PlanAll].Error,
+			expectedError:  errors.New("`plan-all` commands is no longer supported. Use `terragrunt run-all plan` instead."),
 		},
 		{
 			name:           "plan-all with strict mode",
 			controls:       []string{},
 			strictMode:     true,
 			expectedStderr: "",
-			expectedError:  strict.StrictControls[strict.PlanAll].Error,
+			expectedError:  errors.New("is no longer supported"),
```

Comment:
Why is this the expected error now?

Reply:
Every time I add/remove strict control the result is different and I have to edit that test which is annoying.